### PR TITLE
VACMS-2771: Export prod config to code.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #2771

### CODE REVIEW

File: config/sync/core.entity_form_display.node.vamc_operating_status_and_alerts.default.yml [hunk 1]
```diff
@@ -35,8 +35,7 @@ third_party_settings:
       label: Governance
       region: content
     group_editorial_workflow:
-      children:
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.vamc_operating_status_and_alerts.default.yml:39]: @rachel-kauff   Do we know if removing this was intentional ?  (sometimes this field disappears on its own)

File: config/sync/core.entity_form_display.paragraph.q_a_group.default.yml [hunk 2]
```diff
@@ -19,14 +19,14 @@ content:
...
+      open: true
       selection_mode: selection_append
-      field_widget_edit: 0
-      field_widget_replace: 0
-      open: 0
+      field_widget_replace: false
       field_widget_display_settings: {  }
     third_party_settings: {  }
-    type: entity_reference_browser_table_widget
+    type: entity_browser_entity_reference
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.paragraph.q_a_group.default.yml:29]: @rachel-kauff This seems like an unrelated change.  I is it?

File: config/sync/entity_browser.browser.q_a_browser.yml [hunk 3]
```diff
@@ -30,13 +29,3 @@ widgets:
     weight: 1
     label: ''
     id: view
-  72d0ffb9-db92-4f24-ba64-ad72dd2b3bb1:
```
Comment (swirtSJW) [inline config/sync/entity_browser.browser.q_a_browser.yml:33]: Is this removal legit?

Comment (ethanteague) [reply config/sync/entity_browser.browser.q_a_browser.yml:33]: @rachel-kauff | @kevwalsh - is this intentional?

File: config/sync/core.entity_form_display.node.vamc_operating_status_and_alerts.default.yml (hunk 1 above)
Comment (rachel-kauff) [reply config/sync/core.entity_form_display.node.vamc_operating_status_and_alerts.default.yml:39]: @swirtSJW I did not remove any revision log stuff, as far as I know. And if I did, it wasn't intentional.

File: config/sync/field.field.node.vamc_operating_status_and_alerts.field_operating_status_emerg_inf.yml [hunk 4]
```diff
@@ -6,13 +6,19 @@ dependencies:
     - field.storage.node.field_operating_status_emerg_inf
     - node.type.vamc_operating_status_and_alerts
   module:
+    - allowed_formats
     - text
+third_party_settings:
+  allowed_formats:
+    rich_text: '0'
+    rich_text_limited: '0'
```
Comment (swirtSJW) [inline config/sync/field.field.node.vamc_operating_status_and_alerts.field_operating_status_emerg_inf.yml:14]: Was this intentional @rachel-kauff

File: config/sync/core.entity_form_display.node.vamc_operating_status_and_alerts.default.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.node.vamc_operating_status_and_alerts.default.yml:39]: putting back
